lib/ansible/module_utils/cloud/profitbricks/profitbricks.py

```diff
@@ -0,0 +1,105 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
Put this in `lib/ansible/module_utils/profitbricks.py` instead.

Reply:
@mattclay, basically, you are saying I should revert the last commit? Before my last commit `profitbricks.py` was in `lib/ansible/module_utils/`.

Reply:
Yes, although I didn't see the comment from @gundalow earlier that requested the change. We'll need to do something to avoid the conflict. Reverting the change would be the easiest.